### PR TITLE
(doc) Fix JIRA ticket number on known issue.

### PR DESCRIPTION
**Pull request by gguillotte**

### CODE REVIEW

**Comment by gguillotte** [comment]
CC @nathanielksmith @haus

**Comment by nathanielksmith** [comment]
:+1:

File: documentation/release_notes.markdown [hunk 1]
```diff
@@ -43,7 +43,7 @@ Puppet Server 2.3.0 and newer support being restarted by sending a hangup signal
 
 Previous versions of Puppet Server could fail when the `config_version` setting is a single string that contains spaces, which is commonly true when that setting points to a script with space-delineated arguments. Puppet Server 2.3.0 resolves this issue.
 
-* [SERVER-1204](https://tickets.puppetlabs.com/browse/SERVER-1204)
+* [SERVER-1160](https://tickets.puppetlabs.com/browse/SERVER-1160)
```
Comment (rlinehan) [inline documentation/release_notes.markdown]: This is the write ticket number to include here, but I think the description isn't accurate. I've updated SERVER-1160 with some release notes. The description here matches SERVER-1204, but that was never a bug in a released version of Puppet Server.

Comment (rlinehan) [inline documentation/release_notes.markdown]: (sorry I forgot to update the release notes on SERVER-1160 to begin with :( )

**Comment by haus** [comment]
@gguillotte i think in addition to the ticket change the wording should be updated.

Something like...

Previously custom functions or calls to generate could exhaust jrubies. Specifically, this is triggered by calls to commands that block waiting on input (the ticket in question opened a python repl in each jruby). In 2.3.0 this is no longer the case.

**Comment by rlinehan** [comment]
:+1:

**Comment by haus** [comment]
:+1:

**Pull request merged**